tests/openapi/openapi_integration/test_multi_vector.py

```diff
@@ -0,0 +1,199 @@
+                        "max_sim": {}
+                    }
+                }
+            },
+        }
+    )
+    assert response.ok
+
+    response = request_with_validation(
+        api='/collections/{collection_name}',
+        method="GET",
+        path_params={'collection_name': collection_name},
+    )
+    assert response.ok
+
+
+def test_multi_vector_persisted():
+    # batch upsert
+    response = request_with_validation(
+        api='/collections/{collection_name}/points',
+        method="PUT",
+        path_params={'collection_name': collection_name},
+        query_params={'wait': 'true'},
+        body={
+            "points": [
+                {
+                    "id": 1,
+                    "vector": {
+                        "my-multivec": [[0.05, 0.61, 0.76, 0.74]]
+                    }
+                },
+                {
+                    "id": 2,
+                    "vector": {
+                        "my-multivec": [[0.19, 0.81, 0.75, 0.11]]
+                    }
+                },
+                {
+                    "id": 3,
+                    "vector": {
+                        "my-multivec": [[0.36, 0.55, 0.47, 0.94]]
+                    }
+                },
+            ]
+        }
+    )
+    assert response.ok
+
+    # scroll
+    response = request_with_validation(
+        api='/collections/{collection_name}/points/scroll',
+        method="POST",
+        path_params={'collection_name': collection_name},
+        body={"limit": 10, "with_vector": True}
+    )
+    assert response.ok
+    assert len(response.json()['result']['points']) == 3
+    results = response.json()['result']['points']
+
+    first_point = results[0]
+    assert first_point['id'] == 1
+    assert first_point['vector']['my-multivec'] == [[0.05, 0.61, 0.76, 0.74]]
+
+    # retrieve by id
+    response = request_with_validation(
+        api='/collections/{collection_name}/points/{id}',
+        method="GET",
+        path_params={'collection_name': collection_name, 'id': 2},
+    )
+    assert response.ok
+    point = response.json()['result']
+
+    assert point['id'] == 2
+    assert point['vector']['my-multivec'] == [[0.19, 0.81, 0.75, 0.11]]
+
+# TODO add test for failing search with multi vector
+
+def test_multi_vector_validation():
+    # use dense vector
+    response = request_with_validation(
+        api='/collections/{collection_name}/points',
+        method="PUT",
+        path_params={'collection_name': collection_name},
+        query_params={'wait': 'true'},
+        body={
+            "points": [
+                {
+                    "id": 1,
+                    "vector": {
+                        "my-multivec": [0.19, 0.81, 0.75, 0.11]
+                    }
+                }
+            ]
+        }
+    )
+    assert not response.ok
+    assert 'Wrong input: Conversion between multi and regular vectors failed' in response.json()["status"]["error"]
```

Comment:
this is a storage error because we send a `DenseVector` to the `MultiDenseStorage`